youtube_dl/extractor/vvvvid.py

```diff
@@ -259,12 +263,13 @@ def _real_extract(self, url):
                     continue
                 info = self._extract_common_video_info(episode)
                 info.update({
-                    '_type': 'url',
+                    '_type': 'url_transparent',
                     'ie_key': VVVVIDIE.ie_key(),
                     'url': '/'.join([base_url, season_id, video_id]),
                     'title': episode.get('title'),
                     'description': episode.get('description'),
                     'season_id': season_id,
+                    'playlist_title': playlist_title,
```

Comment:
`playlist_title` should be set directly, it should be set in the playlist result title.

Reply:
the show extractor sets the playlist_title according to the playlist each episode belong to.

In the show present as a test for example, in the page are present 3 playlists...
1 - episodes in italian
2 - episodes in japanese
3 - extras
```
λ python -m youtube_dl https://www.vvvvid.it/show/156/psyco-pass -o "%(series)s - %(playlist_title)s - %(episode_number)s - %(title)s" --get-filename
Psycho-Pass - Versione in italiano - 1 - Coefficiente di criminalità
Psycho-Pass - Versione in italiano - 2 - Chi è capace
...
Psycho-Pass - Versione in italiano - 21 - Elogio Insanguinato
Psycho-Pass - Versione in italiano - 22 - Un mondo perfetto
Psycho-Pass - Versione in giapponese - 1 - Coefficiente di criminalità
Psycho-Pass - Versione in giapponese - 2 - Chi è capace
...
Psycho-Pass - Versione in giapponese - 21 - Elogio Insanguinato
Psycho-Pass - Versione in giapponese - 22 - Un mondo perfetto
Psycho-Pass - EXTRA - NA - Mangaka96 - Il Manga
```
As discussed last time the season tag is not the correct tag for this value, nor was the alt_title so the only solution that remains in the `playlist_title` tag which allows the output displayed above.

Reply:
it's about the way you're setting `playlist_title`.
`playlist_title` should not be set directly, it should be set in the playlist result title(by using the `playlist_result` method or using the playlist result type).

Reply:
ok, understood.

I'm looking at creating nested playlists then... like youtube:tab. That con be the correct solution, right?

Reply:
yes

Reply:
ok, done.

My first attemp at using yield... ;-)